imports/client/components/PuzzleListPage.tsx

```diff
@@ -341,55 +342,57 @@ class PuzzleListView extends React.Component<PuzzleListViewProps, PuzzleListView
       }
     }
     const addPuzzleContent = this.props.canAdd && (
-      <div className="add-puzzle-content">
+      <>
         <Button variant="primary" onClick={this.showAddModal}>Add a puzzle</Button>
         <PuzzleModalForm
           huntId={this.props.huntId}
           tags={this.props.allTags}
           ref={this.addModalRef}
           onSubmit={this.onAdd}
         />
-      </div>
+      </>
     );
     return (
       <div>
-        <FormLabel htmlFor="jr-puzzle-search">View puzzles by:</FormLabel>
-        <div className="puzzle-view-controls">
-          <div>
+        <FormGroup>
+          <FormLabel htmlFor="jr-puzzle-search">View puzzles by:</FormLabel>
+          <div className="puzzle-view-controls">
             <ButtonToolbar>
               <ToggleButtonGroup type="radio" className="mr-2" name="puzzle-view" defaultValue="group" value={this.state.displayMode} onChange={this.switchView}>
-                <ToggleButton variant="outline-secondary" value="group">Group</ToggleButton>
-                <ToggleButton variant="outline-secondary" value="unlock">Unlock</ToggleButton>
+                <ToggleButton variant="light" value="group">Group</ToggleButton>
+                <ToggleButton variant="light" value="unlock">Unlock</ToggleButton>
               </ToggleButtonGroup>
               <ToggleButtonGroup
                 type="checkbox"
                 value={this.state.showSolved ? ['true'] : []}
                 onChange={this.changeShowSolved}
               >
-                <ToggleButton variant="outline-secondary" value="true">Show solved</ToggleButton>
+                <ToggleButton variant="light" value="true">Show solved</ToggleButton>
               </ToggleButtonGroup>
             </ButtonToolbar>
+            <ButtonToolbar className="puzzle-list-filter-toolbar">
+              <InputGroup>
+                <FormControl
+                  id="jr-puzzle-search"
+                  as="input"
+                  type="text"
+                  ref={this.searchBarRef}
+                  placeholder="Filter by title, answer, or tag"
+                  value={this.getSearchString()}
+                  onChange={this.onSearchStringChange}
+                />
+                <InputGroup.Append>
+                  <Button variant="danger" onClick={this.clearSearch}>
```

Comment:
Can we swap this to `light` as well?  In red, its loudness on the page greatly exceeds its importance.

Reply:
Or at least `secondary`.